src/getColor/calcAverageColor.ts

```diff
@@ -19,12 +19,10 @@ export const calcAverageColor = (colorBlock: Array<RgbObj>): RgbObj => {
     return colorBlock[0];
   }
 
-  const rgbAverage = colorBlock.reduce((rgbAcc, colorsObj) => {
-    rgbAcc[0] += Math.round(colorsObj.r / totalPixels);
-    rgbAcc[1] += Math.round(colorsObj.g / totalPixels);
-    rgbAcc[2] += Math.round(colorsObj.b / totalPixels);
-    return rgbAcc;
-  }, [0, 0, 0]);
+  const rgbPrimary = colorBlock
+    .map(ar => JSON.stringify(ar))
+    .filter((itm, idx, arr) => arr.indexOf(itm) === idx)
+    .map(str => JSON.parse(str))
```

Comment:
This is a nitpick, and you don't _have_ to do it, but ideally, let's type out the arguments and avoid short names. 🥳

_array, item, index, string_

Reply:
Sure, I'll get that resolved!